app/backplane/power_module/src/main.cpp

```diff
@@ -20,7 +20,7 @@ int main() {
     NRtos::StartRtos();
 
 #ifdef CONFIG_ARCH_POSIX
-    k_sleep(K_SECONDS(10));
+    k_sleep(K_SECONDS(300));
```

Comment:
Why was this changed?

Reply:
We want the sim to last longer than 10 seconds lol. A flight will definitely last longer than 10 seconds. Also, wouldn't have been able to test the 1 minute logging. Unknown why it was set to 10 seconds in the first place